features/story.feature

```diff
@@ -8,6 +8,11 @@ Feature: Story
       | title    | author          | intro                                                                                    |
       | Starship | Mariza & Alfred | Once upon a time, in a far away galaxy. There was a young team on the Discovery Starship |
 
+    And the following user exists
+      | email                       | password |
+      | jenny@ranom.com             | password |
+      | daniel@random.com           | password |
```

Comment:
I can't see that you use these users for anything in this feature. Why do you create them?

Reply:
For the first scenario, testing that a logged in user can access a story